b2sdk/file_version.py

```diff
@@ -282,6 +311,30 @@ def __init__(
             cache_control=cache_control,
         )
 
+    @property
+    def expires(self) -> str | None:
+        return self.file_info.get('b2-expires')
+
+    def expires_parsed(self) -> dt.datetime | None:
+        """Returns the expiration date as a datetime object, or None if there is no expiration date.
+        Raises ValueError if `expires` property is not a valid HTTP-date."""
```

Comment:
there was a discussion recently that we should follow PEP-257, one of the points there is that verbs should be imperative:

>  It prescribes the function or method’s effect as a command (“Do this”, “Return that”), not as a description; e.g. don’t write “Returns the pathname …”.

```suggestion
        """Return the expiration date as a datetime object, or None if there is no expiration date.
        Raise ValueError if `expires` property is not a valid HTTP-date."""
```

Reply:
You're right. I saw a couple of docstrings writting in descriptive manner ("does") and copied that, but now I see that the vast majority of docstrings in b2sdk actually use the imperative style.

Reply:
we can back one at a time :) up to you, not a blocker.